src/main/java/com/terraforged/mod/server/command/TerraCommand.java

```diff
@@ -91,6 +101,18 @@ public static void register(FMLServerStartingEvent event) {
         register(event.getCommandDispatcher());
     }
 
+    @SubscribeEvent
+    public static void register(PlayerEvent.PlayerLoggedInEvent event) {
+        Log.debug("Adding query id entry for {}", event.getPlayer().getDisplayNameAndUUID().getFormattedText());
+        LOCATE_QUERY_IDS.put(event.getPlayer().getUniqueID(), 0);
+    }
+
+    @SubscribeEvent
+    public static void register(PlayerEvent.PlayerLoggedOutEvent event) {
+        Log.debug("Removing query id history for {}", event.getPlayer().getDisplayNameAndUUID().getFormattedText());
```

Comment:
- Not sure how useful this information is to log.
- "id history" sounds a bit grander than what it actually is.
- If it does need logging then it should only do so if a player actually used the command.